### ISSUE TITLE
Should allowedOrigins be an array? 

### ISSUE THREAD

**Issue by bschimpf** [palantir/dropwizard-web-security#42]
Why do we have something that we're just splitting on commas when we could just make it an array and be much clearer?

**Comment by qinfchen** [comment]
I think Jetty's CrossOriginFilter expects it to be a string from `config.getInitParameter`, and we just pass the config to the filter. 

https://github.com/eclipse/jetty.project/blob/master/jetty-servlets/src/main/java/org/eclipse/jetty/servlets/CrossOriginFilter.java#L171

**Comment by bschimpf** [comment]
I get that, I'm wondering if we should provide nice config and assemble the string for the user rather than just blindly using the CrossOriginFilter syntax. I don't have a super strong preference, but this is one of the few places in our ecosystem where multiplicity is specified through bespoke syntax vs. doing it in a true "yaml" setup.

**Comment by derekcicerone-zz** [comment]
When we created this bundle we debated both approaches and decided to favor the native syntax to keep things simple. If you feel strongly I think it makes sense to switch to a list syntax for origins - its one of the few settings for CORS that should actually be tweaked. The CSP will still be in string form though, is that alright?

If we do make any changes here, they need to happen literally today because all Elements apps would need to upgrade.
